core/src/main/resources/operatorsScore-emr-a10G.csv

```diff
@@ -0,0 +1,273 @@
+EndsWith,2.59
+EqualNullSafe,2.59
+EqualTo,2.59
+Exp,2.59
+Explode,2.59
+Expm1,2.59
+First,2.59
+Flatten,2.59
+Floor,2.59
+FormatNumber,2.59
+FromUTCTimestamp,2.59
+FromUnixTime,2.59
+GetArrayItem,2.59
+GetArrayStructFields,2.59
+GetJsonObject,2.59
+GetMapValue,2.59
+GetStructField,2.59
+GetTimestamp,2.59
+GreaterThan,2.59
+GreaterThanOrEqual,2.59
+Greatest,2.59
+HiveGenericUDF,2.59
+HiveSimpleUDF,2.59
+Hour,2.59
+Hypot,2.59
+If,2.59
+In,2.59
+InSet,2.59
+InitCap,2.59
+InputFileBlockLength,2.59
+InputFileBlockStart,2.59
+InputFileName,2.59
+IntegralDivide,2.59
+IsNaN,2.59
+IsNotNull,2.59
+IsNull,2.59
+JsonToStructs,2.59
+JsonTuple,2.59
+KnownFloatingPointNormalized,2.59
+KnownNotNull,2.59
+Lag,2.59
+LambdaFunction,2.59
+Last,2.59
+LastDay,2.59
+Lead,2.59
+Least,2.59
+Length,2.59
+LessThan,2.59
+LessThanOrEqual,2.59
+Like,2.59
+Literal,2.59
+Log,2.59
+Log10,2.59
+Log1p,2.59
+Log2,2.59
+Logarithm,2.59
+Lower,2.59
+MakeDecimal,2.59
+MapConcat,2.59
+MapEntries,2.59
+MapFilter,2.59
+MapKeys,2.59
+MapValues,2.59
+Max,2.59
+Md5,2.59
+MicrosToTimestamp,2.59
+MillisToTimestamp,2.59
+Min,2.59
+Minute,2.59
+MonotonicallyIncreasingID,2.59
+Month,2.59
+Multiply,2.59
+Murmur3Hash,2.59
+NaNvl,2.59
+NamedLambdaVariable,2.59
+NormalizeNaNAndZero,2.59
+Not,2.59
+NthValue,2.59
+OctetLength,2.59
+Or,2.59
+ParseUrl,2.59
+Percentile,2.59
+PercentRank,2.59
+PivotFirst,2.59
+Pmod,2.59
+PosExplode,2.59
+Pow,2.59
+PreciseTimestampConversion,2.59
+PromotePrecision,2.59
+PythonUDF,2.59
+Quarter,2.59
+RLike,2.59
+RaiseError,2.59
+Rand,2.59
+Rank,2.59
+RegExpExtract,2.59
+RegExpExtractAll,2.59
+RegExpReplace,2.59
+Remainder,2.59
+ReplicateRows,2.59
+Reverse,2.59
+Rint,2.59
+Round,2.59
+RowNumber,2.59
+ScalaUDF,2.59
+ScalarSubquery,2.59
+Second,2.59
+SecondsToTimestamp,2.59
+Sequence,2.59
+ShiftLeft,2.59
+ShiftRight,2.59
+ShiftRightUnsigned,2.59
+Signum,2.59
+Sin,2.59
+Sinh,2.59
+Size,2.59
+SortArray,2.59
+SortOrder,2.59
+SparkPartitionID,2.59
+SpecifiedWindowFrame,2.59
+Sqrt,2.59
+Stack,2.59
+StartsWith,2.59
+StddevPop,2.59
+StddevSamp,2.59
+StringInstr,2.59
+StringLPad,2.59
+StringLocate,2.59
+StringRPad,2.59
+StringRepeat,2.59
+StringReplace,2.59
+StringSplit,2.59
+StringToMap,2.59
+StringTranslate,2.59
+StringTrim,2.59
+StringTrimLeft,2.59
+StringTrimRight,2.59
+StructsToJson,2.59
+Substring,2.59
+SubstringIndex,2.59
+Subtract,2.59
+Sum,2.59
+Tan,2.59
+Tanh,2.59
+TimeAdd,2.59
+ToDegrees,2.59
+ToRadians,2.59
+ToUnixTimestamp,2.59
+TransformKeys,2.59
+TransformValues,2.59
+UnaryMinus,2.59
+UnaryPositive,2.59
+UnboundedFollowing$,2.59
+UnboundedPreceding$,2.59
+UnixTimestamp,2.59
+UnscaledValue,2.59
+Upper,2.59
+VariancePop,2.59
+VarianceSamp,2.59
+WeekDay,2.59
+WindowExpression,2.59
+WindowSpecDefinition,2.59
+XxHash64,2.59
+Year,2.59
+Empty2Null,2.59
+WriteFilesExec,2.59
+Ascii,2.59
+ToUTCTimestamp,2.59
+AggregateInPandasExec,1.2
+ArrowEvalPythonExec,1.2
+FlatMapGroupsInPandasExec,1.2
+MapInPandasExec,1.2
+WindowInPandasExec,1.2
+RoundCeil,2.59
+RoundFloor,2.59
```

Comment:
Looks like this is an outdated copy of the score sheet.

missing 4 operators

```
BloomFilterMightContain,
BloomFilterAggregate,
EphemeralSubstring,
KnownNullable,
```

Reply:
Thanks @amahussein! Updated this.